styles.css

```diff
@@ -0,0 +1,232 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600&display=swap");
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.first-card {
+  background-color: #e5e5e5;
+  height: 100vh;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.first-card-section {
+  margin-bottom: 90px;
+}
+
+.all-components {
+  display: grid;
+  grid-row: 2;
+  margin: 16px;
+  background-color: white;
+  padding: 10px;
+  padding-bottom: 60px;
+  border-radius: 16px;
+}
+
+.about-myself-card {
+  background-color: #fff;
+  padding-bottom: 40px;
+  box-sizing: border-box;
+  margin: 0;
+  width: 100%;
+}
+
+.all-components .about-myself-card {
+  border-top-right-radius: 120px;
+}
+
+.first-image {
+  width: 300px;
+  height: 260px;
+  margin: 0 auto;
+  margin-top: 10px;
+}
+
+.tonic-content {
+  margin-top: -10px;
+}
+
+.tonic-content h2 {
+  font-size: 34px;
+  margin-bottom: 10px;
+}
+
+ul li {
+  margin-top: 5px;
+  margin-bottom: 8px;
+  color: rgb(95, 95, 173);
+}
+
+.tonic-list li {
+  font-size: 13px;
+  color: #7a869a;
+  line-height: 16px;
+  margin-left: 25px;
+  margin-top: -2px;
+}
+
+.tonic-content ul li {
+  list-style: none;
+}
+
+ul {
+  display: flex;
+}
+
+.tonic-flex ul {
+  display: flex;
+  flex-direction: column;
+}
+
+.tonic-content-canopy {
+  display: flex;
+  align-items: baseline;
+  width: 300px;
+  margin-top: 10px;
+}
+
+p {
+  margin-bottom: 15px;
+  color: #344563;
+  font-size: 15px;
+  line-height: 24px;
+  line-height: 26px;
+}
+
+.tonic-content-canopy p {
+  color: #344563;
+  position: static;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 16px;
+}
+
+h1 {
+  font-size: 36px;
+  margin-top: -20px;
+}
+
+.btn-tonic {
+  padding: 10px;
+  width: 140px;
+  font-size: 18px;
+  background-color: white;
+  border: 1px #396df2 solid;
+  border-radius: 8px;
+  color: #396df2;
+  cursor: pointer;
+  margin-top: 20px;
+}
+
+.btn-tonic:hover {
+  background-color: #ebebff;
+  color: black;
+}
+
+.tonic-courses ul li {
+  list-style: none;
+  margin-right: 10px;
+}
+
+.tonic-courses ul li a {
+  margin-top: 20px;
+  text-decoration: none;
+  color: #6070ff;
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+}
+
+.second-card-section {
+  margin-bottom: 90px;
+}
+
+.third-card-section {
+  margin-bottom: 90px;
+}
+
+.fourth-card-section {
+  margin-bottom: 90px;
+}
+
```

Comment:
- [ ] Good job 👏  on following the original template design however you still need some last improvements on your design. Make your card projects have a fixed width (`width: 327px`) and add a border around the card just like the original design is(`border: 1px solid #DFE1E6`).